OPHD/UI/NotificationWindow.h

```diff
@@ -26,4 +33,6 @@ class NotificationWindow : public Window
 	Button btnTakeMeThere{ "Take Me There" };
 	TextArea mMessageArea;
 	bool mTakeMeThereHidden{ false };
+
+	Signal mTakeMeThereClicked;

```

Comment:
Wondering if we should use a present tense for this?
`mTakeMeThereClick`

Reply:
Well, considering that the action has already happened (e.g., the button has been clicked, the mouse button was let go of), it seems appropriate.

I'm not really opposed to a present tense but I don't know if there's really a value in that.

Reply:
A lot of the event handler functions were updated to use present tense. It might be more consistent if everything used present tense.

If you think about it, it's not really until the event handler code has run that the event is really complete. So the point where that object is relevant, the event is still live being processed.

Reply:
I mean, you bring up a good point. I'll have to adjust the way I'm thinking about it. I looked into some of the .Net code for C# GUI applications and they follow the present tense format as well so I'll start to conform to thinking in the present tense as well. Might take a bit of time to adjust though! :D